newsletters/008.md

```diff
@@ -15,25 +18,40 @@ Most of the code is already open-source and can be obtained from repositories su
 
 https://github.com/sophgo/linux-riscv
 
++ Feature enhancement
++ Reduce bootargs
+
 ### U-Boot
 
 https://github.com/sophgo/u-boot/tree/sg2042-dev
 
++ No commits this week
+
 ### OpenSBI
 
 https://github.com/sophgo/opensbi/tree/sg2042-dev
 
++ No commits this week
+
 ## Case Study
 
 We're looking for fun, good, or profitable use cases for SG2042. Feel free to share your experiences with us - [just send a PR!](https://github.com/sophgocommunity/SG2042-Newsletter/pulls)
 
 ## Events and Games
 
++ [The open-source AI Compiler Development Course is now online!](https://mp.weixin.qq.com/s/S0fCMEDcivE3z0u81CYIoA)
++ [Open Source｜The 1st "SOPHON RISC-V in Colleges · Software Freedom Day" Event is Now Open!](https://mp.weixin.qq.com/s/ajCygefU9lSEy84UnE5whQ)
+
 
 ## In the News
 
-+ [Compiling image file for Milk-V Duo on Linux｜Video Tutorial](https://www.bilibili.com/video/BV1em4y1P718/?spm_id_from=333.999.0.0&vd_source=ca24cf8467ac967e45410d8de9072922)
++ [[Milk-V Duo] Serial Console Login Attempt | RJ45 Network Tutorial | Access the system terminal via USB - Part 1](https://b23.tv/rv5EP8V)
++ [[Milk-V Duo] Serial Console Login Attempt | RJ45 Network Tutorial | Access the system terminal via USB - Part 2](https://b23.tv/fph5dBq)
++ [[Milk-V Duo] Controlling an External LED Blinking](https://b23.tv/Xc5RwON)
++ [Compiling image for Milk-V Duo on Linux｜Video Tutorial](https://www.bilibili.com/video/BV1em4y1P718/?spm_id_from=333.999.0.0&vd_source=ca24cf8467ac967e45410d8de9072922)
++ [Deploying WebAssembly runtime environment and controlling LED on Milk-V Duo](https://community.milkv.io/t/duo-webassembly-udp/603)
++ [Milk-V Duo Free Trial - SPI Driver Auto Transmit and Receive Code](https://bbs.elecfans.com/jishu_2375983_1_1.html)

```

Comment:
`Implementing full-duplex communication using SPI driver` could be a better description

Reply:
Thank you for your advice. It has been revised ; )